### PR TITLE
Name vars

### PR DESCRIPTION
**Pull request by peterschubert**
make the distinction between different names and codes used by the service

### CODE REVIEW

File: cookiecutter.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
-  "project_name": "UPP Golang microservice template",
-  "service_name": "{{ cookiecutter.project_name.lower().replace(' ', '-') }}",
-  "project_short_description": "UPP Golang microservice template",
+  "app_name": "Golang Microservice Template",
+  "service_name": "{{ cookiecutter.app_name.lower().replace(' ', '-') }}",
+  "system_code": "upp-{{ cookiecutter.service_name }}",
```
Comment (tosan88) [inline cookiecutter.json]: Nice!

File: {{ cookiecutter.repo_name }}/main.go [hunk 2]
```diff
@@ -11,19 +11,21 @@ import (
...
-	app := cli.App("{{ cookiecutter.service_name }}", "{{ cookiecutter.project_short_description }}")
+	app := cli.App("{{ cookiecutter.service_name }}", appDescription)
 
 	appSystemCode := app.String(cli.StringOpt{
 		Name:   "app-system-code",
-		Value:  "{{ cookiecutter.service_name }}",
+		Value:  "{{ cookiecutter.system_code }}",
 		Desc:   "System Code of the application",
 		EnvVar: "APP_SYSTEM_CODE",
 	})
```
Comment (iulianp26) [inline {{ cookiecutter.repo_name }}/main.go]: Perhaps serviceName should be got too as it will be probably used more on logging.

**Comment by tamas-molnar** [comment]
IMO the

- repo name
- app name
- service name
- system code
- docker image name

should all be the same, for consistency and simplicity.

File: cookiecutter.json (hunk 1 above)
Comment (sjwells) [inline cookiecutter.json]: I think the dewey system code should NOT have the upp prefix. The dewey system name should. See e.g. https://systemregistry.in.ft.com/manage/aggregate-concept-transformer

**Pull request merged**